Source/Core/DolphinQt/NetPlay/NetPlayDialog.h

```diff
@@ -7,6 +7,7 @@
 #include <QDialog>
 
 #include "Core/NetPlayClient.h"
+#include "UICommon/GameFile.h"
```

Comment:
`GameFile` should be able to be a forward declaration here, with the include being able to be moved to the cpp file.